README.md

```diff
@@ -1512,6 +1513,44 @@ would enable the use of a `@fold` on the `adjacent_animal` vertex field of `Foo`
 }
 ```
 
+### SchemaGraph 
+The `SchemaGraph` is an utility class that encodes a database schema and allows for better schema 
+introspection. The  `SchemaGraph` has three main benefits over the GraphQL schema. First, 
+it's able to store and expose a schema's index information. Second, it can correctly represent the 
+inheritance of non-abstract classes. Finally, it exposes many utility functions, such as `
+get_subclass_set`, that make it easier to explore the schema.
+
+We plan to add `SchemaGraph` generation from SQLAlchemy metadata and can currently generate a 
+`SchemaGraph` from OrientDB schema metadata as exemplified by the following mock example. 
+
+```python
+from graphql_compiler.schema_generation.orientdb.schema_graph_builder import (
+    get_orientdb_schema_graph
+)
+from graphql_compiler.schema_generation.orientdb.utils import (
+    ORIENTDB_INDEX_RECORDS_QUERY, ORIENTDB_SCHEMA_RECORDS_QUERY
+)
+
+# Get schema metadata from hypothetical Animals database.
+client = your_function_that_returns_an_orientdb_client()
+schema_records = client.command(ORIENTDB_SCHEMA_RECORDS_QUERY)
+schema_data = [x.oRecordData for x in schema_records]
```

Comment:
I would probably opt for `record` instead of `x` since this is meant to be read.